### PR TITLE
Add timrjs

### PR DESCRIPTION
**Pull request by joesmith100**

### CODE REVIEW

File: files/timrjs/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "timrjs",
+  "repo": "joesmith100/timrjs",
+  "files": {
+    "basePath": "dist/",
+    "include": ["timr.js", "timr.min.js"]
```
Comment (jimaek) [inline files/timrjs/update.json]: Just timr.min.js is enough here

Comment (joesmith100) [inline files/timrjs/update.json]: @jimaek I want to include both files though?

Comment (jimaek) [inline files/timrjs/update.json]: You need that in production?

Comment (joesmith100) [inline files/timrjs/update.json]: No, but the dev version contains more descriptive errors. So would be nice to be able to offer the option to people.

**Pull request merged**